src/main/groovy/wooga/gradle/build/unity/ios/IOSBuildPluginExtension.groovy

```diff
@@ -17,6 +17,7 @@
 
 package wooga.gradle.build.unity.ios
 
+import com.sun.org.apache.xpath.internal.operations.Bool
```

Comment:
That looks like a rogue auto import

Reply:
Stupid IDEA. Don't pause and think of the type is `boolean` or `bool` for the current language ;)